storage/remote/intern.go

```diff
@@ -40,13 +40,17 @@ type pool struct {
 }
 
 type entry struct {
-	// Keep all 64bit atomically accessed variables at the top of this struct.
-	// See https://golang.org/pkg/sync/atomic/#pkg-note-BUG for more info.
-	refs int64
+	refs atomic.Int64
 
 	s string
 }
 
+func newEntry(s string) *entry {
+	entry := &entry{s: s}
+	entry.refs.Store(1)
```

Comment:
I would expect a `newEntry` to have 0 refs in the returned object. So let's skip storing 1 here (while keeping the method) and increment where `newEntry` is called.

Reply:
Addressed in 9ef9045